packages/xo-server/src/api/vdi.mjs

```diff
@@ -66,6 +66,14 @@ export const set = defer(async function ($defer, params) {
 
     await xapi.resizeVdi(ref, size)
   }
+  if('cbt' in params){
+    if(params.cbt === true){
+      await xapi.VDI_enableChangeBlockTracking(ref)
+    }
+    if(params.cbt === false){
+      await xapi.VDI_disableChangeBlockTracking(ref) 
+    }
+  } 
```

Comment:
@fbeauchamp told me that he prefers to avoid this because it gets very difficult to find if a method is used or not, and I agree with him.

@fbeauchamp what do you think of this?

```suggestion
  if('cbt' in params){
    await xapi[params.cbt ? 'VDI_enableChangeBlockTracking' : 'VDI_disableChangeBlockTracking'](ref)
  } 
```

Reply:
I am not sure of the benefits over a real call that is refactorable
```js
await  params.cbt ? xapi.VDI_disableChangeBlockTracking(ref)  : xapi.VDI_enableChangeBlockTracking(ref) 
```